.github/workflows/ci.yml

```diff
@@ -55,7 +59,17 @@ jobs:
           python -m pip install cumulusci
 
       - name: Run Tests
-        run: python -m pytest
+        run: |
+          python -m coverage run -m pytest
+
+      - name: Check Coverage
+        run: |
+          coverage xml
+          coverage report
+          echo $BASE_REF
+          git diff HEAD..$BASE_REF
+          diff-cover coverage.xml --fail-under 100 --compare-branch=$BASE_REF --diff-range-notation=.. --show-uncovered --markdown-report coverage.md
```

Comment:
Looks like `--compare-branch` defaults to `origin/main`, so `$BASE_REF` isn't necessary.